### DISCUSSION TITLE
"Add new field" suggest to have multiple select option

### DISCUSSION THREAD

**Discussion by CHTeow22** [community/community#10017] [Projects and Issues]
<img width="317" alt="Screenshot 2565-01-12 at 12 11 50 PM" src="https://user-images.githubusercontent.com/24906304/149062581-cdfe6ca2-a390-4339-af9f-295fbcff7f91.png">
For example I want to add new field named "project", the issue may involve in a few project, such as frontend (UI design) & backend (function). 

For now I only can select one of them, cannot have both.

**Comment by harisjaliawala** [comment]
We have the same issue. Would be nice to have a multiple select type field!

**Reply by coderbyheart** [reply]
See https://github.com/github/feedback/discussions/6580
